HW02_A_Graph-Fails.Rmd

```diff
@@ -51,36 +68,65 @@ ggplot(data = mpg) +
 ### Graph Fail 3
 Two mistakes in this graph. First, I wanted to make the the points slightly bolder, but changing the alpha to 2 does nothing. What does alpha do and what does setting it to 2 do? What could be done instead if I want the points slightly bigger? 
 
+**My answer/comment:**
+
+*alpha is for degree of transparency of the points, which ranges from 0(transparent) to 1(opaque). Using "size=2" can make the points bolder.*

```

Comment:
I am curious what "alpha=2" does to the plot exactly. My guess is that alpha=2 is equivalent to alpha=1. But I didn't find any documentation about the effect of alpha>1...

Reply:
I tried this alph=2, it seems same results with alpha=1 by eyeballing it.